### DISCUSSION TITLE
How do I create new root folders in the browser?

### DISCUSSION THREAD

**Discussion by funkychimp** [frntc/Sidekick64#79] [Q&A]
Hi,

How do I create new root folders in the browser? That is, when I open the sidekick64 browser I can only see the default SD card root folders such as CRT, PRG, KERNAL Etc.  Is there away to create new folders in the browser such as 'Demos' or Favorites' for example.

Thanks in advance

Paul

**Comment by frntc** [comment]
Hi,

currently these default folders are hard-coded. But of course you can create sub-folders inside them.

**Comment by funkychimp** [comment]
Hi frntc,

Thank you for the quick repsonse. Yea I currently have my own folders in the CRT folder. I have one for 'Demos' and  one for 'Fav Games'. It would be nice if the browser could see them if placed in the Sds root but it is what it is. 

Thanks

Paul
